voyager-publish/index.js

```diff
@@ -29,8 +29,10 @@ export const publish = async ({
   // Fetch the count of all unpublished measurements - we need this for monitoring
   // Note: this number will be higher than `measurements.length` because voyager-api adds more
   // measurements in between the previous and the next query.
+  // Note: counting ALL rows can put too much load on the DB server when the table is very large.
+  // Let's stop after we count 10 million rows. That's enough to let us know that we are in a serious trouble.
   const totalCount = (await pgPool.query(
-    'SELECT COUNT(*) FROM measurements'
+    'SELECT COUNT(*) FROM (SELECT 1 FROM measurements LIMIT 10000000) t;'
```

Comment:
```suggestion
    'SELECT COUNT(*) FROM (SELECT 1 FROM measurements LIMIT 10000000);'
```

I think that should fix the test

Reply:
```
postgres=# SELECT COUNT(*) FROM (SELECT 1 FROM measurements LIMIT 10000000);
ERROR:  subquery in FROM must have an alias
LINE 1: SELECT COUNT(*) FROM (SELECT 1 FROM measurements LIMIT 10000...
                             ^
HINT:  For example, FROM (SELECT ...) [AS] foo.
```

Reply:
ah, then maybe the js side needs to be adjusted